main.js

```diff
@@ -66,7 +66,7 @@ function run() {
 	let rx = x - warp.x;
 	let ry = y - warp.y;
 	let heading = Math.round((Math.atan2(-rx, ry) * 1800 / Math.PI)) / 10;
-	str += "The location is " + Math.round(warp.dist2(x, y)) + " blocks away from the warp, at a heading of " + heading + "°.<br>\n";
+	str += "The location is " + Math.round(warp.dist2(x, y)) + " blocks away from the warp, at a heading of " + heading + "&deg;.<br>\n";
```

Comment:
Chrome displays e.g. `…heading of 73.2Â°…` when using `°`.
By using the according named entity `&deg;`, Chrome will display `…heading of 73.2°.…`